en/deploy-on-general-kubernetes.md

```diff
@@ -124,7 +124,7 @@ Different clusters can be in the same or different `namespace`, which is based o
 
 ## Initialize the TiDB cluster
 
-If you want to initialize your cluster after deployment, refer to [Initialize a TiDB Cluster in Kubernetes](initialize-a-cluster.md).
```

Comment:
`Deploy TiDB in General Kubernetes` -> `Deploy TiDB on General Kubernetes`

https://github.com/ran-huang/docs-tidb-operator/blob/on-k8s/en/deploy-multiple-tidb-operator.md?plain=1#L58

```
2. Refer to [Deploy TiDB on General Kubernetes](deploy-on-general-kubernetes.md) to deploy the TiDB cluster.
```